.storybook/webpack.config.js

```diff
@@ -1,15 +1,28 @@
-const path = require('path');
+const path = require('path')
 
-module.exports = (baseConfig, env, defaultConfig) => {
-  // Extend defaultConfig as you need.
-
-  // For example, add typescript loader:
-  defaultConfig.module.rules.push({
-    test: /\.(js|jsx)$/,
-    loaders: ['eslint-loader'],
-    exclude: [/node_modules/],
-    include: path.resolve(__dirname, '../'),
-  });
-
-  return defaultConfig;
-};
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        include: path.resolve(__dirname, '../src/css/'),
```

Comment:
would rather output the prod css to `./dist` and then include that here so we know we are testing/using production styles that have gone through compilation

Reply:
How are we supposed to dev then? The styles are still going through the same build pipeline so they shouldn't be any different.
We could change the build output to use `dist/` instead of `build/`